docs/installation-guide/src/main/asciidoc/installing.adoc

```diff
@@ -660,49 +565,49 @@ Table 1-2 {productName} Full Profile and Web Profile Features
 |===
 |Feature |Full Platform |Web Profile
 
-|Java Platform, Enterprise Edition 8 |X |X
+|Jakarta EE 10 Platform |X |X
 
-|Jakarta EE Security API |X |X
+|Jakarta Security 3.0 supporting OpenID Connect |X |X
 
-|Java API for JSON Processing 1.1 |X |X
+|Jakarta JSON Processing 2.1.3 |X |X
```

Comment:
I would remove the patch version, which can easily change with some fix. THe major and minor version of components is important.

```suggestion
|Jakarta JSON Processing 2.1 |X |X
```